src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -560,6 +560,8 @@ type_to_extract_op (MonoTypeEnum type)
 }
 
 static guint16 sri_vector_methods [] = {
+	SN_Add,
```

Comment:
Unrelated, but why aren't these implemented in managed code using
if (Arm64.IsSupported) ... etc.

Reply:
Good question. This whole work was motivated by #49397. My understanding of the purpose of this work is to minimize the managed code that .NET customers need to author.